enhancements/machine-api/azure-ultra-disks.md

```diff
@@ -170,7 +215,126 @@ const (
 
 ### Implementation Details/Notes/Constraints
 
-N/A
+#### Initializing Data Disks 
+While one of the goals for this enhancement is to provide automation for creating and attaching Azure Ultra Disks as Data Disks to Machines, there is one caveat.
+Once an Azure instance is created and a Data Disk (in this case of type Ultra Disk) is attached to said instance, the Disk will be seen as a _raw_ block device by the OS.
+
+This means that in order for the Data Disk to be usable by workloads it will require some form of initialization, which, given the high performing nature of Ultra Disks (and more generally of Data Disks) is best suited to be left to the cluster administrator.
+This way they will be able to decide how to best slice the disk in partitions, while also choosing how to format them and autonomously deciding where to have them mounted.
+
+For this reason, guidance will be provided on how a cluster administrator can specify flexible configuration to achieve so with the help of a supporting example.
```

Comment:
this makes it sound like the guidance will be provided somewhere else, but it looks like the following instructions are the guidance. if this is the guidance we are delivering, i would change this sentence to indicate that. for example:

```suggestion
For this reason, the following example is provided as guidance on how a cluster administrator can specify flexible configurations for their Data Disks:
```

Reply:
Makes sense, thanks. Included.